src/components/ha-date-range-picker.ts

```diff
@@ -179,12 +172,31 @@ export class HaDateRangePicker extends LitElement {
 
       paper-input {
         display: inline-block;
-        max-width: 200px;
+        max-width: 250px;
+        min-width: 200px;
+        --paper-input-container-shared-input-style_-_font-size: 14px;
```

Comment:
Let's only make the font smaller on smaller screens

Reply:
Don't we use 14 everywhere else anyway?

Reply:
Then it should not be specified I guess?

Reply:
We use 14 (mostly) the default in this element is 16

Reply:
Where is that default coming from? I dont we think we set `paper-input-container-shared-input-style_-_font-size` in other places?

Reply:
It originates here <https://github.com/PolymerElements/paper-styles/blob/master/classes/typography.js#L117>

Reply:
That's subhead? We don't use that here do we?

Reply:
I just checked, all our inputs are 16px, our body font size is 14px